crates/turbopack-core/src/context.rs

```diff
@@ -9,34 +10,66 @@ use crate::{
     source::Source,
 };
 
+#[turbo_tasks::value(shared)]
+pub enum ProcessResult {
+    Module(Vc<Box<dyn Module>>),
+    Ignore,
+}
```

Comment:
`AssetContext::process` returns this enum now. Instead of returning a module it might ignore the reference. This won't include the module at that place.

It's common that callers actually need a Module, so there is a `ProcessResult::module()` method which gives the module and throws otherwise.